src/plugins/intel_cpu/tests/functional/single_layer_tests/conversion.cpp

```diff
@@ -68,6 +68,42 @@ class ConvertCPULayerTest : public testing::WithParamInterface<convertLayerTestP
 
         function = makeNgraphFunction(ngPrc, params, conversion, "ConversionCPU");
     }
+
+    void generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticShapes) override {
+        if (outPrc != Precision::BOOL) {
+            SubgraphBaseTest::generate_inputs(targetInputStaticShapes);
+            return;
+        }
+
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+
+        auto shape = targetInputStaticShapes.front();
+        ov::Tensor tensor;
+        tensor = ov::test::utils::create_and_fill_tensor(funcInputs[0].get_element_type(), shape);
```

Comment:
create_and_fill_tensor has additional arguments that allows to set filling range. Is it possible to use them?

Reply:
Fixed. Range is set by create_and_fill_tensor now. By the way, start_from and resolution are still applied outside this function to get more diversified floating point value. Thanks Dmitry!